src/commands/Owner/rp.ts

```diff
@@ -27,6 +30,21 @@ export default class extends BotCommand {
 		[cmd, input, str]: [string, KlasaUser | undefined, string | undefined]
 	) {
 		if (msg.guild!.id !== '342983479501389826') return null;
+
+		switch (cmd.toLowerCase()) {
+			case 'resettask': {
+				const { currentTask } = await getUsersCurrentSlayerInfo(msg.author.id);
+				if (!currentTask) return msg.channel.send(`You have no task.`);
+				await currentTask.remove();
+				return msg.channel.send(`Deleted your task.`);
+			}
+			case 'slayerreset': {
+				await SlayerTaskTable.delete({ user: await getNewUser(msg.author.id) });
+				await msg.author.settings.reset(UserSettings.Skills.Slayer);
+				return msg.channel.send(`Deleted ALL your task history, your slayer XP`);
+			}
+		}
```

Comment:
This would be usable in the live bot! Time to delete these

Reply:
or just put them in hax if youd want them for later